src/main/webapp/disable.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019-2020 The MathWorks, Inc.
+// This script file is used to disable all web elements of depricated MATLAB build step.
+
+
+//Disable all previous UI elements.
+var testMode = getElementsByXpath("//td[contains(text(),'Test mode')]/../td[3]/select");
+disableAllElements(testMode);
+
+var taPDFReportChkBx = getElementsByXpath("//input[@name='taPDFReportChkBx']");
+disableAllElements(taPDFReportChkBx);
+
+var tatapChkBx = getElementsByXpath("//input[@name='tatapChkBx']");
+disableAllElements(tatapChkBx);
+
+var taJunitChkBx = getElementsByXpath("//input[@name='taJunitChkBx']");
+disableAllElements(taJunitChkBx);
+
+var taSTMResultsChkBx = getElementsByXpath("//input[@name='taSTMResultsChkBx']");
+disableAllElements(taSTMResultsChkBx);
+
+var taCoberturaChkBx = getElementsByXpath("//input[@name='taCoberturaChkBx']");
+disableAllElements(taCoberturaChkBx);
+
+var taModelCoverageChkBx = getElementsByXpath("//input[@name='taModelCoverageChkBx']");
+disableAllElements(taModelCoverageChkBx);
+
+
+//Function to disable all occurrences of given web element.
+function disableAllElements(nodeName){
+  var a =[];
+  a.push(nodeName.iterateNext());

```

Comment:
I think the structure of this code can be simplified (sorry for wrong js syntax)


node = nodeName.IterateNext();
while((node  !== null) {
  a.push(node);
 node = nodeName.IterateNext();
}
Thus the array doesn't need a terminating null because size of array is available.
The second while loop should just loop on the number of elements in the array.

Reply:
Done.